command/plugin_info_test.go

```diff
@@ -102,6 +103,52 @@ func TestPluginInfoCommand_Run(t *testing.T) {
 		}
 	})
 
+	t.Run("version flag", func(t *testing.T) {
+		t.Parallel()
+
+		pluginDir, cleanup := vault.MakeTestPluginDir(t)
+		defer cleanup(t)
+
+		client, _, closer := testVaultServerPluginDir(t, pluginDir)
+		defer closer()
+
+		const pluginName = "azure"
+		_, sha256Sum := testPluginCreateAndRegister(t, client, pluginDir, pluginName, consts.PluginTypeCredential, "v1.0.0")
+
+		for name, tc := range map[string]struct {
+			version     string
+			expectedSHA string
+		}{
+			"versioned":       {"v1.0.0", sha256Sum},
+			"builtin version": {versions.GetBuiltinVersion(consts.PluginTypeSecrets, pluginName), ""},
+		} {
+			t.Run(name, func(t *testing.T) {
+				ui, cmd := testPluginInfoCommand(t)
+				cmd.client = client
+
+				code := cmd.Run([]string{
+					"-version=" + tc.version,
+					consts.PluginTypeCredential.String(), pluginName,
+				})
+
+				combined := ui.OutputWriter.String() + ui.ErrorWriter.String()
+				if exp := 0; code != exp {
```

Comment:
I left it out of the test case struct because it doesn't vary from one test case to another. It would be a reasonable addition in the future though if someone wanted to add a test case command that is expected to fail.

Reply:
That's what it looked like. 👍 Thanks for the clarification!